net/http.go

```diff
@@ -77,3 +79,90 @@ type DoHTTPRequest interface {
 	// SetCookies adds cookies to the jar
 	SetCookies(url *url.URL, cookies []*http.Cookie)
 }
+
+// NewHTTPRedirectClient is the default net/http client with headers being set on redirect
```

Comment:
Is this interface implementation being used somewhere in core or connect?

Reply:
Well, to be honest, I don't even know how this ended up here, I must've accidentally pull something as it's not present in master: https://github.com/beaconsoftwarellc/gadget/blob/master/net/http.go

But there are 4 occurrences where `NewHTTPRedirectClient` is used: https://github.com/search?q=org%3Aecatholic+NewHTTPRedirectClient&type=code

Maybe I'll just remove it?

Reply:
Looks like it was removed as deprecated code in commit: cb2c2f0f6425cdb52838b0b1856b67ba74443854